docs/source/user-guide/integrations/h2o.rst

```diff
@@ -0,0 +1,171 @@
+Using deepchecks with an H2O Model
+==================================
+
+`H2O <https://docs.h2o.ai/h2o/latest-stable/h2o-docs/automl.html>`__ is
+a wildly used AutoML framework, written in Java but with multiple fully
+implemented api libraries, including one in python. In this example,
+we’ll be showing how you can effortlessly test your H2O model with
+deepchecks in the exact same way you can test any sklearn model.
+
+Building an H2O Model
+---------------------
+
+First we’ll load our dataset, using the adult dataset that can be
+downloaded from the `UCI machine learning
+repository <http://archive.ics.uci.edu/ml>`__ and is also available
+in `deepchecks.tabular.datasets`.
+
+.. code:: ipython3
+
+    import warnings
+    warnings.filterwarnings('ignore')
```

Comment:
This is in order for the output to be clearer and without warnings?
Wondering if we maybe need to say something about this.

Reply:
If the code is not actually running (e.g. we're not displaying output), we don't need the filter.